data/campaigns/The_South_Guard/scenarios/06a_Tidings_Good_and_Ill.cfg

```diff
@@ -365,7 +365,7 @@
             message= _ "Ho! Me see elves. Elves tasty? Me take elves to eat."
         [/message]
         [message]
-            race=elf
+            speaker=second_unit
```

Comment:
Some have different words for certain objects depending upon context and who's speaking, too. It's not too hard to run through and look for single/multiple and first/second/third person as well as male/female. But for the rest, the translators need to send feedback to the programmers for a change.